controllers/territory_controller/territory_controller.py

```diff
@@ -58,6 +58,8 @@ def __init__(self, record_arena_actions: bool) -> None:
         }
 
         self._log: List[Tuple[StationCode, Claimant, float]] = []
+        # starting with a dirty log ensures the structure is written for every match
```

Comment:
Really minor - would be great to have comments as proper sentences:
```suggestion
        # Starting with a dirty log ensures the structure is written for every match.
```